.eslintrc.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
 
   // Stop ESLint from looking for a configuration file in parent folders
   root: true,
+  env: {
+    es2020: 'true',
```

Comment:
What are the implications of this setting? Are we going to accidentally allow more things in other source files which we aren’t meant to use (because we aim to be compatible with older browsers)?

Reply:
You're right there could be many more implications that I didn't consider. I got rid of that change and added the comment /* global BigInt */ to the RulesOfHooks.js to get rid of the eslint error.